.circleci/config.yml

```diff
@@ -35,15 +35,14 @@ jobs:
             bash -c "comm -13 <(sort /tmp/store-path-pre-build | grep -v '\.drv$') <(nix path-info --all | grep -v '\.drv$' | sort) | cachix push $CACHIX_NAME"
   build-mac:
     macos:
-      xcode: 11.3.0
+      xcode: 13.2.0
     environment:
       CACHIX_NAME: vulcanclimatemodeling
     steps:
       - run:
           name: Install Nix
           command: |
-            curl -k https://curl.se/ca/cacert.pem -o ~/.cacert.pem
-            export CURL_CA_BUNDLE=~/.cacert.pem
+            echo "" >> $HOME/.bash_profile
```

Comment:
Workaround for a strange issue: the CircleCI runner's .bash_profile doesn't have a newline at the end of the file, so after `nix` adds the line to source its config, the file fails to evaluate.